test/psbt.spec.ts

```diff
@@ -97,8 +97,67 @@ describe('Psbt', () => {
         assert.strictEqual(tx.toHex(), f.transaction);
       });
     });
-  });
 
+    it('should keep the proofs inside witness utxos when encode/decode base64 confidential psbt.', () => {
+      const psbt = new Psbt();
+
+      const witnessUtxo = {
+        script: Buffer.from(
+          '76a91439397080b51ef22c59bd7469afacffbeec0da12e88ac',
+          'hex',
+        ),
+        asset: Buffer.from(
+          '5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225',
+          'hex',
+        ),
```

Comment:
```suggestion
        asset: Buffer.concat([Buffer.alloc(1,1), Buffer.from(
          '5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225',
          'hex',
        ).reverse()]),
```

To parse an asset from string to bytes, it must be converted, reversed and it needs to be prefixed with a `0x01` uncofidential prefix.

Reply:
Ok I just saw that you're mocking a confidential utxo.
In this case, asset and value MUST be commitments starting with either `0x08` or `0x09`, you're using unblinded data instead.

Reply:
Owh It works :) thanks